src/app/shared/components/addscenario/addscenario.component.ts

```diff
@@ -42,8 +43,9 @@ export class AddScenarioModal implements OnInit, OnDestroy {
     public cloneParameters: any;
     public regRegion: any;
     public statisticGroup: any;
-    public clone: boolean;
+    public clone: boolean = false;
```

Comment:
This is fine, but for future reference you don't need to type set it if you're setting an initial value. So just saying `public clone = false;` would have been fine :)